scripts/irods/test/test_iquest.py

```diff
@@ -150,6 +150,13 @@ def test_iquest_incorrect_format_count(self):
         self.admin.assert_icommand("iquest \"%s %s\" \"select COLL_NAME where COLL_NAME like '%home%'\"",
                                    'STDERR_SINGLELINE', 'boost::too_few_args: format-string referred to more arguments than were passed')
 
+    def test_iquest_does_not_fail_to_find_data_objects_with_SQL_keyword_names_select_and_where__issue_7174(self):
+        for data_object in ['selected_file', 'where_file', 'select.txt', 'where.txt']:
```

Comment:
Could use `unittest.subTest` for each of these. Up to you

Reply:
Updated to use subtest.